src/sql/workbench/parts/notebook/cellViews/textCell.component.ts

```diff
@@ -152,16 +170,34 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 				this._content = this.cellModel.source;
 			}
 
-			this._commandService.executeCommand<string>('notebook.showPreview', this.cellModel.notebookModel.notebookUri, this._content).then((htmlcontent) => {
-				htmlcontent = this.convertVscodeResourceToFileInSubDirectories(htmlcontent);
-				htmlcontent = this.sanitizeContent(htmlcontent);
+			if (this.useInProcMarkdown) {
+				let uri = Object.create(null);
```

Comment:
The markdown renderer allows special handling for uri's if they're already known. Since we don't know them at this point, that's why we're creating an object with no properties here.

export interface IMarkdownString {
	value: string;
	isTrusted?: boolean;
	uris?: { [href: string]: UriComponents };
}

Reply:
OK, I think we could inline this or since it's optional just not fill in?

Reply:
Ack won't fill it in.